pyquil/operator_estimation.py

```diff
@@ -810,11 +811,71 @@ def serializable(self):
         }
 
 
+def generate_experiment_programs(tomo_experiment: TomographyExperiment,
+                                 active_reset: bool = False) \
+        -> Tuple[List[Program], List[List[int]]]:
+    """
+    Generate the programs necessary to estimate the observables in a TomographyExperiment.
+    Grouping of settings to be run in parallel, e.g. by a call to group_experiments, should be
+    done before this method is called.
+
+    .. CAUTION::
+        One must be careful with compilation of the output programs before the appropriate MEASURE
+        instructions are added, because compilation may re-index the qubits so that
+        the output list of `measure_qubits` no longer accurately indexes the qubits that
+        should be measured.
+
+    :param tomo_experiment: a single TomographyExperiment to be translated to a series of programs
+        that, when run serially, can be used to estimate each of obs_expt's observables.
```

Comment:
what is `obs_expt`? parameter that disappeared in the refactoring?

Reply:
Yeah, good catch, it should be tomo_experiment.

```suggestion
        that, when run serially, can be used to estimate each of its observables.
```